### PR TITLE
[Coq] utf8 notations in Coq export

### PR DESCRIPTION
**Pull request by olaure01**
Experimental notations in Coq for readability of proof script executions.

### CODE REVIEW

File: export_as_coq.ml [hunk 1]
```diff
@@ -15,9 +15,9 @@ let proof_to_coq proof =
     let conclusion = get_conclusion proof in
     let header = "(* This Coq file has been generated using the C1ick ⅋ c⊗LLec⊥ tool. *)\n"
         ^ "(* https://click-and-collect.linear-logic.org/ *)\n"
-        ^ "(* First download and install NanoYalla version 1.0.2, see README.md: *)\n"
+        ^ "(* First download and install NanoYalla version 1.0.3, see README.md: *)\n"
         ^ "(* https://click-and-collect.linear-logic.org/download/nanoyalla.zip *)\n\n" in
-    let start_file_line = "From NanoYalla Require Import macroll.\n\nSection TheProof.\n\n" in
+    let start_file_line = "From NanoYalla Require Import macroll.\nImport LLNotations.\n\n\nSection TheProof.\n\n" in
```
Comment (olaure01) [inline export_as_coq.ml:20]:
```suggestion
    let start_file_line = "From NanoYalla Require Import macroll.\n\nImport LLNotations.\n\nSection TheProof.\n\n" in
```

**Comment by olaure01** [comment]
Could we have it in pre-prod to see how it behaves? and I would be happy to have comments on your side about whether it is useful or not.
If everything works as planned, removing the line `Import LLNotations.` in the Coq file should give exactly the same behavior as before (for users who prefer it).

**Comment by etiennecallies** [comment]
I can deploy it on preprod as soon as the PR https://github.com/etiennecallies/click-and-collect/pull/99 is merged.

**Comment by etiennecallies** [comment]
Deployed on preprod. For the moment I don't see the difference...

**Comment by olaure01** [comment]
I am trying to avoid the explicit `%nat` in the Coq export but failing so far (see coq/coq#14305).

**Pull request merged**